src/test/java/kitchenpos/helper/OrderHelper.java

```diff
@@ -0,0 +1,52 @@
+package kitchenpos.helper;
+
+
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderType;
+
+import java.util.List;
+import java.util.UUID;
+
+public final class OrderHelper {
+
+    private OrderHelper() {
+    }
+
+    public static Order createOrderTypeIsDelivery(OrderType orderType, List<OrderLineItem> orderLineItems, String deliveryAddress) {
+        Order order = new Order();
+        order.setId(UUID.randomUUID());
+        order.setType(orderType);
+        order.setOrderLineItems(orderLineItems);
+        order.setDeliveryAddress(deliveryAddress);
+        return order;
+    }
```

Comment:
메서드에 이미 배달주문건을 생성한다는 의미여서 orderType을 파라미터를 안받아도 되지 않을까 생각이 들어요.
아래처럼 바꾸는 것은 어떤가요?
```suggestion
public static Order createOrderTypeIsDelivery(List<OrderLineItem> orderLineItems, String deliveryAddress) {
        Order order = new Order();
        order.setId(UUID.randomUUID());
        order.setType(OrderType.DELIVERY);
        order.setOrderLineItems(orderLineItems);
        order.setDeliveryAddress(deliveryAddress);
        return order;
    }
```